src/app/machines/views/MachineList/MachineListTable/MachineListTable.tsx

```diff
@@ -846,8 +846,36 @@ export const MachineListTable = ({
     [hiddenColumns, showActions]
   );
 
+  const getMachinesDisplayed = () => {
```

Comment:
Might be an overkill, but the original name could be a bit misleading.
```suggestion
  const getCurrentPageDisplayedMachineCount = () => {
```

Reply:
Please add tests which clearly explain the expected behaviour (we expect to have the original page size returned unless you're on the last page).
This will be easier to test if you move this function (or entire component as suggested below) to a separate file.